core/sdk-core/src/test/java/software/amazon/awssdk/core/internal/metrics/BytesSentTrackingPublisherTest.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.core.internal.metrics;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.reactivex.Flowable;
+import java.nio.ByteBuffer;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.reactivestreams.Publisher;
+import software.amazon.awssdk.core.SdkRequest;
+import software.amazon.awssdk.core.SdkRequestOverrideConfiguration;
+import software.amazon.awssdk.core.http.NoopTestRequest;
+import software.amazon.awssdk.core.internal.progress.listener.DeafultProgressUpdater;
+import software.amazon.awssdk.core.internal.util.UploadProgressUpdaterInvocation;
+import software.amazon.awssdk.core.progress.listener.ProgressListener;
+
+public class BytesSentTrackingPublisherTest {
+
+    @Test
+    public void test_updatesBytesSent() {
```

Comment:
`test_` prefix is really not needed. Let's follow our unit tests naming pattern

Reply:
changed